skimage/measure/_regionprops_utils.py

```diff
@@ -326,3 +327,38 @@ def perimeter_crofton(image, directions=4):
 
     total_perimeter = coefs @ h
     return total_perimeter
+
+
+def _normalize_spacing(spacing, ndims):
```

Comment:
It's worth noting that this function currently let's `np.nan` and `np.inf` pass! It seems to me that these don't really make sense as a value for `spacing`. My instinct is raise a `ValueError` for these non-rational cases as well. What do you think?

Reply:
Yes. That makes sense.

Reply:
Added the checks in _normalize_spacing